### ISSUE TITLE
Annotation service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled not working on NLB

### ISSUE THREAD

**Issue by luisamador** [kubernetes/cloud-provider-aws#168]
**What happened**:
*service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"*
The annotation above doesn't take any effect on NLBs. It doesn't matter if you are updating an existing NLB or if you are provisioning it from scratch.

**What you expected to happen**:
The "Cross-zone load balancing" setting enabled on the NLB.

**How to reproduce it (as minimally and precisely as possible)**:
Just create a K8s Service resource with the following annotations:
```
service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
service.beta.kubernetes.io/aws-load-balancer-internal: "true"
service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
```

**Environment**:
both Kubectl and AWS EKS cluster on version v1.18.9 

/kind bug

**Comment by mars64** [comment]
I think you're looking for the [`aws-load-balancer-controller` project](https://github.com/kubernetes-sigs/aws-load-balancer-controller) -- [they just took over for the Ticketmaster/CoreOS ALB Ingress Controller](https://github.com/kubernetes/enhancements/issues/423#issuecomment-726253356) and I think are the ones that [provide the annotation you reference](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.1/guide/service/annotations/).

**Comment by kishorj** [comment]
Legacy cloud provider has the fix https://github.com/kubernetes/kubernetes/pull/95247, and will be available in 1.20.
